src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -455,6 +455,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._register(this._model.contentChanged((change) => this.handleContentChanged(change)));
 		this._register(this._model.onProviderIdChange((provider) => this.handleProviderIdChanged(provider)));
 		this._register(this._model.kernelChanged((kernelArgs) => this.handleKernelChanged(kernelArgs)));
+		this._register(this._model.onActiveCellChanged(() => this.detectChanges()));
```

Comment:
Are there other places where we call detectChanges in this file on active cell being changed? I'd like to avoid calling `detectChanges()` multiple times when active cell is changed.

If we're just reacting to this event coming from the model, can we remove other calls to `detectChanges()`?